php-checks/src/main/java/org/sonar/php/checks/formatting/ControlStructureSpacingCheck.java

```diff
@@ -75,15 +74,15 @@ private void checkSpaceForStatement(FormattingStandardCheck formattingCheck, Ast
     for (AstNode semicolon : node.getChildren(PHPPunctuator.SEMICOLON)) {
       Token semicolonToken = semicolon.getToken();
       Token nextToken = semicolon.getNextAstNode().getToken();
-      int nbSpace = getNbSpaceBetween(semicolonToken, nextToken);
+//      int nbSpace = getNbSpaceBetween(semicolonToken, nextToken);
 
-      if (nbSpace != 1 && isOnSameLine(semicolonToken, nextToken)) {
-        shouldReportIssue = true;
-      }
+//      if (nbSpace != 1 && isOnSameLine(semicolonToken, nextToken)) {
+//        shouldReportIssue = true;
+//      }
     }
 
     if (shouldReportIssue) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Change this condition so that it does not always evaluate to "false" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2583)